rollup.config.js

```diff
@@ -0,0 +1,35 @@
+import nodeResolve from 'rollup-plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+import babel from 'rollup-plugin-babel';
+import uglify from 'rollup-plugin-uglify';
+
+export default {
+  input: './source/index.js',
+  output: {
+    file: 'dist/umd/react-virtualized.js',
+    format: 'umd',
+    name: 'ReactVirtualized',
+    globals: {
+      react: 'React',
+      'react-dom': 'ReactDOM',
+    },
+  },
+  external: ['react', 'react-dom'],
+  plugins: [
+    nodeResolve(),
+    commonjs({
+      include: 'node_modules/**',
+    }),
+    babel({
+      exclude: 'node_modules/**',
+      runtimeHelpers: true,
```

Comment:
would be good to compare actual output for this and for the variant with `plugins: ['external-helpers']` in case of umd build, I suspect that latter might be slightly better or there is no difference - im curious which one of those 2 😅